src/test/kotlin/testhelper/mock/PdlMock.kt

```diff
@@ -17,10 +17,10 @@ import testhelper.getRandomPort
 
 fun PersonIdentNumber.toHistoricalPersonIdentNumber(): PersonIdentNumber {
     val firstDigit = this.value[0].digitToInt()
-    val newDigit = firstDigit + 4
+    val lastDigitOfNewNumber = (firstDigit + 4) % 10
```

Comment:
Hva med å heller endre teste-data til å være et mer gyldig fødelsnummer?

Forresten, så er tanken her at det første sifferet i fødelsnummeret får plusset på 4 og dermed blir et D-nummer.
Ser at koden ikke ble helt som tenkt, men ser ut som det blir riktig som tenkt om man setter inn dette i stedet:
```
val dNummer = this.value.replaceRange(
        startIndex = 0,
        endIndex = 0,
        replacement = newDigit.toString(),
    )
```

Reply:
Jeg kan godt endre testdata, ja. Koden fungerer som tenkt, men kun med gyldige fnr, så det vil ikke hjelpe å presisere index. Lederen i testdata har `9` som første siffer, og da prøver man å legge på "13" foran resten av strengen, og får en karakter for mye.